src/mongocrypt-marking.c

```diff
@@ -916,6 +945,256 @@ _mongocrypt_fle2_placeholder_to_find_ciphertext (
    return res;
 }
 
+static bool
+isInfinite (bson_iter_t iter)
+{
+   if (!BSON_ITER_HOLDS_DOUBLE (&iter)) {
+      return false;
+   }
+   return isinf (bson_iter_double (&iter));
```

Comment:
`bson_iter_double()` already returns zero if the value is not a double.

Reply:
Oh nice. Removed the `BSON_ITER_HOLDS_DOUBLE` condition.